internal/clients/connect/client_connect_test.go

```diff
@@ -568,3 +568,86 @@ func (s *ConnectClientSuite) TestTestAuthenticationNotPublisher() {
 	s.NotNil(err)
 	s.ErrorContains(err, "user account bob with role 'viewer' does not have permission to publish content")
 }
+
+func (s *ConnectClientSuite) TestGetContent() {
+	httpClient := &http_client.MockHTTPClient{}
+	created, err := time.Parse(time.RFC3339, "2024-09-05T22:18:02Z")
+	if err != nil {
+		s.Fail("Unable to parse Created Time")
+	}
+	deployed, err := time.Parse(time.RFC3339, "2024-09-05T22:18:02Z")
+	if err != nil {
+		s.Fail("Unable to parse Deployed Time")
+	}
+	httpClient.On("Get", "/__api__/v1/content", mock.Anything, mock.Anything).Return(nil).RunFn = func(args mock.Arguments) {
+		contentList := args.Get(1).(*[]connectGetContentDTO)
+
+		*contentList = append(*contentList, connectGetContentDTO{
```

Comment:
Since the underlying HTTP client overwrites the argument, you should probably leave out the `append` call and instead do 
```
*contentList = connectGetContentDTO{
...
```